CONTRIBUTING.md

```diff
@@ -0,0 +1,49 @@
+# Contributing
+
+We welcome contributions to this project. Please follow our code of conduct (CODE_OF_CONDUCT).
+
+To make code changes you will need to fork the project. Thereafter follow the set up instructions in the README file.
+
+If you don't want to make changes yourself, but would like to make a suggestion or comment, you can open an issue.
+
+## Opening an issue
+
+We recommend opening an issue for the following situations:
+
+* Reporting an error you can’t solve yourself
+* Discussing a high-level topic or idea (for example, community, vision or policies)
+* Proposing a new feature or other project idea
+
+This list is taken from [Open Source Guides](https://opensource.guide/how-to-contribute/#how-to-submit-a-contribution).
+
```

Comment:
I suggest we add another heading here for "Opening a pull request" and add the situations list from the [Open Source Guides](https://opensource.guide/how-to-contribute/#opening-a-pull-request) for consistency with the Issues section.

Reply:
There is Creating a pull request later for those who want to contribute in code.

Reply:
Yes, I am just missing a heading here to mark the transition to describing the PR process :) 

Also, I think it would be helpful to explicitly define the situations in which a PR should be considered (the "You should usually open a pull request in the following situations: ..." part from the Guide) to use the same approach as with Issues -- and to avoid having to reject bigger PRs that were not agreed upon ahead of time.

Reply:
Fair enough, good points. I've added an 'Opening a pull request' section. Although the list of steps is a little out of place, I think it is helpful for those unfamiliar with the process (they can find out more about the details of each step from e.g. the egghead videos).